.github/workflows/release-swift-toolchain-binary-sizes.yml

```diff
@@ -48,9 +48,10 @@ jobs:
     strategy:
       matrix:
         include:
-          - os: windows-latest
-            arch: amd64
-          # TODO(thebrowsercompany/swift-build/issues/129): Support arm64 releases.
+          - arch: amd64
+            os: windows-latest
+          - arch: arm64
+            os: cirun-win11-23h2-pro-arm64-16-2024-04-22
```

Comment:
It's fine for testing but ultimately you'll want the kill switch to work too.

Reply:
Yes that is a good point. I'll send another PR to mirror the "USE_CIRUN"  feature you added in our primary repo